** M-x report-emacs-bug includes the Debian version.
   Provided by: Romain Francoise <rfrancoise@debian.org>

** M-x report-emacs-bug CCs the report to the maintainer.
   Provided by: Romain Francoise <rfrancoise@debian.org>

--- emacs-snapshot-20070902.orig/lisp/mail/emacsbug.el
+++ emacs-snapshot-20070902/lisp/mail/emacsbug.el
@@ -98,7 +98,7 @@
 	  (with-current-buffer (get-buffer-create "*Messages*")
 	    (point-max-marker)))
     (compose-mail reporting-address
-		  topic)
+		  topic '(("Cc" . "rfrancoise@debian.org")))
     ;; The rest of this does not execute
     ;; if the user was asked to confirm and said no.
     (rfc822-goto-eoh)
@@ -146,6 +146,11 @@
     (add-text-properties (1+ user-point) (point) prompt-properties)
 
     (insert "\n\nIn " (emacs-version) "\n")
+    (insert " (Debian emacs-snapshot package, version "
+	    (substring (shell-command-to-string
+			"dpkg -s emacs-snapshot-common|grep ^Version:")
+		       9 -1)
+	    ")\n")
     (if (fboundp 'x-server-vendor)
 	(condition-case nil
             ;; This is used not only for X11 but also W32 and others.
